src/boards/esp01/board.mk

```diff
@@ -32,30 +32,31 @@ INC += $(SIMBA_ROOT)/src/boards/esp01
 SRC += $(SIMBA_ROOT)/src/boards/esp01/board.c
 
 LINKER_SCRIPT ?= simba.flash.512k.ld
-ESP_FLASH_SIZE = 512K
+ESP_FLASH_SIZE = 512KB
 
 BOARD_HOMEPAGE = "http://espressif.com"
 BOARD_PINOUT = "esp01-pinout.png"
 BOARD_DESC = "ESP-01"
 
 MCU = esp8266
 SERIAL_PORT ?= /dev/arduino
-BOARD_PY = $(SIMBA_ROOT)/src/boards/esp12e/board.py
-RUN_PY ?= $(SIMBA_ROOT)/src/boards/esp12e/run.py
+BOARD_PY = $(SIMBA_ROOT)/src/mcus/$(MCU)/board.py
+RUN_PY ?= $(SIMBA_ROOT)/make/run.py
 CONSOLE_RESET_TYPE ?= 0
 TIMEOUT ?= 10
 BAUDRATE ?= 76800
 
 upload:
```

Comment:
The _upload_ and _rerun_ recipes are also common for all esp8266 boards, so it may be worth to move them into one common place too. But maybe later, in other PR.

Reply:
Agreed, lets make changes in small steps.